src/main/java/dev/jbang/dependencies/DependencyUtil.java

```diff
@@ -39,6 +39,7 @@
 
 	static {
 		aliasToRepos = new HashMap<>();
+		aliasToRepos.put("mavencentral", "https://repo1.maven.org/maven2/");
 		aliasToRepos.put("jbossorg", "https://repository.jboss.org/nexus/content/groups/public/");
 		aliasToRepos.put("redhat", "https://maven.repository.redhat.com/ga/");
 		aliasToRepos.put("jcenter", "https://jcenter.bintray.com/");
```

Comment:
this one you're keeping for now?

Reply:
Yeah. Don't break those who used it explicitly.

Reply:
Yeah. Should we perhaps show a warning to people using it?

Reply:
not sure where to best put that - we can do that in separate PR if we really feel for it.

Reply:
Perhaps right where we parse //REPOS?